datapacks/Bookshelf/data/bs.random/.metadata/module.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "bs.random",
+    "display_name": "Random",
+    "description": "Contains many random functions for different distributions",
+    "documentation": "https://bookshelf.docs.gunivers.net/en/latest/modules/random.html",
+    "authors": [
+        "SBtree"
+    ],
+    "contributors": [
+        "Aeldrion"
+    ]
```

Comment:
Since the author and contributor are already specified in at least one feature metadata, you don't have to define them here. Indeed, our scripts can extract this information from the feature tag metadata.
In practice, the authors/contributors part of the `module.json` is mainly for modules where we have features that are not functions. For instance, the `bs.tree` module is just a bunch of structures and we cannot define the authors of these structures inside them since they are not text files.